conf/modules/post_variant_calling.config

```
@@ -17,6 +17,7 @@
 process {
 
     withName: 'GERMLINE_VCFS_CONCAT'{
+        ext.args   = { -a }
```

not sure why the tests are failing, but maybe it is missing quotes:

```suggestion
        ext.args   = { "-a" }
```

---

Ah yes, stupid mistake when quickly reimplementing something I had manually configured using the Seqera Platform.